HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -5,27 +5,30 @@ categories:: Debugging
 You can inspect much of the internal structure of the class library and other data structures.
 This can often be useful for research and debugging purposes.
 
-warning:: The keyboard shortcuts mentioned here only applies to the Mac SC.app. This document should probably be updated. ::
+note:: The keyboard shortcuts mentioned here only applies to the standard supecollider IDE.::
 
 section:: Class Definitions, Implementations, and References
 
-Selecting the name of any Class (e.g. Object) and typing cmd-j will open its class definition file.
+subsection::cmd-i
```

Comment:
ok. what are the default shortcuts for the three OSes? (that you remapped yours is arguably your own fault :))